triggers/daml/Daml/Trigger.daml

```diff
@@ -1,130 +1,164 @@
 -- Copyright (c) 2019 The DAML Authors. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
-{-# LANGUAGE AllowAmbiguousTypes #-}
+
 daml 1.2
 module Daml.Trigger
-  ( Message(..)
```

Comment:
Apparently git doesn’t realize this but `Daml.Trigger.LowLevel` is almost a 1:1 copy of the previous state of `Daml.Trigger`. The only changes are a bunch of exports (e.g. `Command` turns out to be useful for handling `commandsInFlight`), a few instances (e.g. `Ord` for `CommandId`) and an optional `CommandId` field on transaction.